winml/api/Windows.AI.MachineLearning.idl

```diff
@@ -33,7 +33,7 @@ import "windows.storage.idl";
 
 namespace ROOT_NS.AI.MachineLearning 
 {
-    [contractversion(4)]
+    [contractversion(5)]
```

Comment:
Please note that an associated change is required in the OS repo to indicate that the contract version has been bumped.

Reply:
Could you give more details what change need to done in OS repo?

---
In reply to: [536308141](https://github.com/microsoft/onnxruntime/pull/5877#discussion_r536308141) [](ancestors = 536308141)